Cargo.toml

```diff
@@ -18,9 +18,10 @@ path = "benches/crit_bench.rs"
 
 [dev-dependencies]
 criterion = "0.3.3"
-lz4 = "1.23.1"
+lzzzz = "0.8"
```

Comment:
why did you change the lz4 to lzzzz?

Reply:
lzzzz supports all lz4(C) features, including the frame format. I need that to have a baseline implementation to compare against.